tldw_Server_API/app/core/Setup/install_manager.py

```
@@ -442,8 +474,382 @@ def execute_install_plan(plan_payload: dict[str, Any]) -> None:
+
+
+def _qualify_install_step_name(
+    name: str,
+    plan: InstallPlan,
+    *,
+    bundle_id: str,
+    resource_profile: str,
+    catalog_version: str,
+) -> str:
+    selection_key = build_audio_selection_key(bundle_id, resource_profile, catalog_version)
+    if name.startswith("deps:") or name.startswith("embeddings:") or name == "stt:silero_vad":
```

Ruff (0.15.6)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)